src/pages/Community/PostDetail.js

```
@@ -72,344 +85,755 @@ const PostDetail = () => {
+                `${process.env.REACT_APP_LOCAL_BACKEND_COMMUNITY_URI}/post/${teamParam}/${postId}`,
+                {withCredentials: true}
+            );
+            let postData = {...res.data, id: res.data.postId || res.data.id};
 
-                if (commentId === undefined || commentId === null) {
-                    return null;
+            // User#숫자 처리
+            const match = postData.writerNickname && postData.writerNickname.match(/^User#(\d+)$/);
+            if (match) {
+                const writerId = match[1];
+                postData._writerId = writerId;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)